subsys-srr.typ

```diff
@@ -0,0 +1,67 @@
+
+They will also be equipped with solar panels and rechargeable batteries in order to keep the low amount of power they need for a very extended amount of time.
+
+=== On-Base Antenna
+
+The largest prohibitor of range is the curvature of the planet, and the higher a transceiver is, the further it can send a message.
+
+A secondary component to combat this issue that will be sent with the colonists is another transceiver that will be mounted atop the colony at the highest point. This antenna will allow messages to be sent from the colony to people travelling at a much greater range than just using the handheld device of around 15-20 miles.
+
+There is one pull back to this method to be aware of: The longer range of sending messages is exclusively for sending from the colony and not receiving. The height of the transceiver does not affect the range of messages being received, only messages sent. 
+
+We still believe it is worth bringing the transceiver because even if there is not two-way communication at a longer distance, the colonists can still direct people out on mission about critical information even if they venture beyond the 6 mile radius. 
+
+=== Radio Methodology
+
+The radio waves we will be using are Ultra-High Frequency, or UHF, waves. These are frequencies in the range of 300 megahertz to 3 gigahertz and are regularly used on Earth for radio walkie-talkie communications.
+
+These waves travel the best in line-of-sight applications, and hold a better accuracy of data transmission as compared to lower frequencies.
+
+A further, and much more critical concern, with using traditional lower-range high frequency, or HF, waves is that they work by bouncing off the ionosphere of Earth to travel long distance. The ionosphere of Mars is thinner, and will not perform nearly as well as it will on Earth. Mars is also covered in a layer of iron, which will greatly impede the radio transmission distance and accuracy of the transmission using these waves.
+
+As a result, the UHF waves provide the best middle-ground of being compatible with the environment and being very easy to set up and use.
+
+
+== Interactions with Other Systems
+
+=== Standalone System
+
+The SRR is designed to be a standalone system with no interdependencies on the other communications systems. If all else were to go down, colonists will still be able to communicate amongst themselves and conduct short-range missions. This is intentional to act as a safeguard so that there is never a total communication outage on the colony.
+
+== Integration and Deployment
+
+There is a two-step process to the SRR:
+- Each colonist carries one handheld radio communication device.
+- The colony base is equipped with a dedicated transceiver for a longer range of transmission.
+
+The figure below showcases the range differences and how this will exist on the colony.
```

Comment:
nit: reference the figure
```suggestion
@Figure::colony-radio-range below showcases the range differences and how this will exist on the colony.
```